Requirement already satisfied: torchaudio in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (2.5.0)
Requirement already satisfied: torch==2.5.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torchaudio) (2.5.0)
Requirement already satisfied: filelock in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (3.16.1)
Requirement already satisfied: typing-extensions>=4.8.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (4.12.2)
Requirement already satisfied: networkx in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (3.4.1)
Requirement already satisfied: jinja2 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (3.1.4)
Requirement already satisfied: fsspec in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (2024.9.0)
Requirement already satisfied: setuptools in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (69.5.1)
Requirement already satisfied: sympy==1.13.1 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from torch==2.5.0->torchaudio) (1.13.1)
Requirement already satisfied: mpmath<1.4,>=1.1.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from sympy==1.13.1->torch==2.5.0->torchaudio) (1.3.0)
Requirement already satisfied: MarkupSafe>=2.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from jinja2->torch==2.5.0->torchaudio) (3.0.1)
Note: you may need to restart the kernel to use updated packages.
Requirement already satisfied: pydub in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (0.25.1)
Note: you may need to restart the kernel to use updated packages.
Requirement already satisfied: soundfile in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (0.12.1)
Requirement already satisfied: cffi>=1.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from soundfile) (1.17.1)
Requirement already satisfied: pycparser in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from cffi>=1.0->soundfile) (2.22)
Note: you may need to restart the kernel to use updated packages.
Requirement already satisfied: wandb in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (0.18.5)
Requirement already satisfied: click!=8.0.0,>=7.1 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (8.1.7)
Requirement already satisfied: docker-pycreds>=0.4.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (0.4.0)
Requirement already satisfied: gitpython!=3.1.29,>=1.0.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (3.1.43)
Requirement already satisfied: platformdirs in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (4.2.2)
Requirement already satisfied: protobuf!=4.21.0,!=5.28.0,<6,>=3.19.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (5.28.3)
Requirement already satisfied: psutil>=5.0.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (5.9.0)
Requirement already satisfied: pyyaml in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (6.0.2)
Requirement already satisfied: requests<3,>=2.0.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (2.32.3)
Requirement already satisfied: sentry-sdk>=2.0.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (2.17.0)
Requirement already satisfied: setproctitle in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (1.3.3)
Requirement already satisfied: setuptools in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from wandb) (69.5.1)
Requirement already satisfied: six>=1.4.0 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from docker-pycreds>=0.4.0->wandb) (1.16.0)
Requirement already satisfied: gitdb<5,>=4.0.1 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from gitpython!=3.1.29,>=1.0.0->wandb) (4.0.11)
Requirement already satisfied: charset-normalizer<4,>=2 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from requests<3,>=2.0.0->wandb) (3.4.0)
Requirement already satisfied: idna<4,>=2.5 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from requests<3,>=2.0.0->wandb) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from requests<3,>=2.0.0->wandb) (2.2.3)
Requirement already satisfied: certifi>=2017.4.17 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from requests<3,>=2.0.0->wandb) (2024.8.30)
Requirement already satisfied: smmap<6,>=3.0.1 in /opt/anaconda3/envs/TD6/lib/python3.12/site-packages (from gitdb<5,>=4.0.1->gitpython!=3.1.29,>=1.0.0->wandb) (5.0.1)
Note: you may need to restart the kernel to use updated packages.
[34m[1mwandb[0m: [33mWARNING[0m Calling wandb.login() after wandb.init() has no effect.
shape of waveform torch.Size([1, 110250]), sample rate with 22050, label is 3
shape of spectogram torch.Size([1, 201, 552])
Waveform: tensor([[ 0.0345,  0.0482,  0.0643,  ..., -0.1070, -0.0598, -0.0341]])

Forma de los datos: torch.Size([128, 1, 110250])
Etiquetas: tensor([2, 1, 7, 3, 6, 2, 3, 1, 0, 0, 6, 0, 9, 3, 3, 7, 9, 6, 6, 2, 4, 9, 7, 9,
        3, 6, 0, 2, 1, 9, 0, 9, 6, 5, 5, 5, 7, 1, 0, 7, 5, 6, 0, 9, 8, 5, 4, 6,
        1, 9, 2, 8, 7, 7, 9, 2, 3, 7, 0, 5, 1, 8, 3, 9, 1, 6, 6, 8, 2, 0, 5, 2,
        6, 1, 2, 5, 2, 1, 6, 5, 8, 8, 0, 9, 0, 9, 5, 8, 4, 6, 9, 4, 2, 8, 0, 2,
        6, 8, 9, 9, 3, 5, 2, 1, 1, 1, 2, 8, 5, 6, 1, 9, 1, 2, 4, 7, 2, 2, 7, 5,
        5, 0, 7, 8, 8, 8, 9, 0])
MLP(
  (fc1): Linear(in_features=110250, out_features=1024, bias=True)
  (fc2): Linear(in_features=1024, out_features=512, bias=True)
  (fc3): Linear(in_features=512, out_features=256, bias=True)
  (fc4): Linear(in_features=256, out_features=10, bias=True)
  (dropout): Dropout(p=0.4, inplace=False)
)
Number of parameters: 113555722
Epoch: [1/10], Train loss: 2.2807
Epoch: [1/10], Valid loss: 2.2471, Valid accuracy: 0.1212
Saving best model at epoch 1 with valid_loss 2.2471
/opt/anaconda3/envs/TD6/lib/python3.12/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/opt/anaconda3/envs/TD6/lib/python3.12/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
Epoch: [2/10], Train loss: nan
Epoch: [2/10], Valid loss: 2.2486, Valid accuracy: 0.1111
Epoch: [3/10], Train loss: nan
Epoch: [3/10], Valid loss: 2.2004, Valid accuracy: 0.1414
Saving best model at epoch 3 with valid_loss 2.2004
Epoch: [4/10], Train loss: nan
Epoch: [4/10], Valid loss: 2.2036, Valid accuracy: 0.1414
Epoch: [5/10], Train loss: nan
Epoch: [5/10], Valid loss: 2.2173, Valid accuracy: 0.2020
Epoch: [6/10], Train loss: nan
Epoch: [6/10], Valid loss: 2.2524, Valid accuracy: 0.1818
Epoch: [7/10], Train loss: nan
Epoch: [7/10], Valid loss: 2.3270, Valid accuracy: 0.2020
Epoch: [8/10], Train loss: nan
Epoch: [8/10], Valid loss: 2.4363, Valid accuracy: 0.1818
Epoch: [9/10], Train loss: nan
Epoch: [9/10], Valid loss: 2.6080, Valid accuracy: 0.1919
Epoch: [10/10], Train loss: nan
Epoch: [10/10], Valid loss: 2.7628, Valid accuracy: 0.2323
